### PR TITLE
[JENKINS-66445] Only print a warning if file from system property doe…

### PR DESCRIPTION
**Pull request by Dohbedoh**
…s not exist

[JENKINS-66445](https://issues.jenkins.io/browse/JENKINS-66445)

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: src/main/java/com/cloudbees/jenkins/support/filter/ContentMappings.java [hunk 1]
```diff
@@ -162,14 +162,15 @@ private ContentMappings(@Nonnull XmlProxy proxy) {
...
         Set<String> words = new HashSet<>();
-        String fileLocation = System.getProperty(ADDITIONAL_STOP_WORDS_PROPERTY, SupportPlugin.getRootDirectory()
-            + "/" + ADDITIONAL_STOP_WORDS_FILENAME);
-        if (fileLocation != null) {
-            LOGGER.log(Level.FINE, "Attempting to load user provided stop words from ''{0}''.", fileLocation);
-            File f = new File(fileLocation);
-            if (!f.exists() || !f.canRead()) {
-                LOGGER.log(Level.WARNING, "Could not load user provided stop words as " + fileLocation 
-                    + " does not exist or is not readable.");
+        String fileLocationFromProperty = System.getProperty(ADDITIONAL_STOP_WORDS_PROPERTY);
```
Comment (MRamonLeon) [inline src/main/java/com/cloudbees/jenkins/support/filter/ContentMappings.java:165]: Why did you split this into two more complex sentences? I prefer the simplest way if there is no need to change it.

Comment (Dohbedoh) [reply src/main/java/com/cloudbees/jenkins/support/filter/ContentMappings.java:165]: If the system property is used, but the path provided does not exist, we want to log a warning.
If the default file location does not exist, we do not want to log a warning.
In both cases those, the logic will be the same. There is just a particular scenario (system property used but file not exist where we want to log something).
Does that make sense ?

File: src/main/java/com/cloudbees/jenkins/support/filter/ContentMappings.java [hunk 2]
```diff
@@ -179,10 +180,12 @@ private ContentMappings(@Nonnull XmlProxy proxy) {
                     }
                     return words;
                 } catch (IOException ex) {
-                    LOGGER.log(Level.WARNING, "Could not load user provided stop words as " + fileLocation 
+                    LOGGER.log(Level.WARNING, "Could not load user provided stop words as " + fileLocation
```
[the remark this answers is not in the document]
Comment (Dohbedoh) [reply src/main/java/com/cloudbees/jenkins/support/filter/ContentMappings.java:183]: I see and I agree.

File: src/main/java/com/cloudbees/jenkins/support/filter/ContentMappings.java [hunk 3]
```diff
@@ -179,10 +180,12 @@ private ContentMappings(@Nonnull XmlProxy proxy) {
...
                         + " does not exist or is not readable.", ex);
```
Comment (Dohbedoh) [inline src/main/java/com/cloudbees/jenkins/support/filter/ContentMappings.java:184]:
```suggestion
                    LOGGER.log(Level.WARNING, "Could not load user provided stop words. there was an error reading " + fileLocation, ex);
```

File: src/main/java/com/cloudbees/jenkins/support/filter/ContentMappings.java (hunk 1 above)
Comment (MRamonLeon) [reply src/main/java/com/cloudbees/jenkins/support/filter/ContentMappings.java:165]: I see now, thanks

**Pull request merged**